gui/default/index.html

```diff
@@ -871,6 +871,14 @@ <h4 class="panel-title">
                           </span>
                         </td>
                       </tr>
+                      <tr ng-if="connections[deviceCfg.deviceID].connected && (deviceCfg.numConnections > 1 || connections[deviceCfg.deviceID].secondary)">
+                        <th><span class="fas fa-fw fa-random"></span>&nbsp;<span translate>Number of Connections</span></th>
+                        <td class="text-right">
+                          <span>
```

Comment:
I don't think the empty `span` is needed here.

Reply:
Improved